spec/mailers/admin_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe AdminMailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Are you planning to add tests here?

Reply:
yes, of course, i will create separate PR for specs